src/Analyzers/Core/CodeFixes/GenerateConstructor/AbstractGenerateConstructorService.State.cs

```diff
@@ -238,13 +238,7 @@ private bool TryInitializeDelegatedConstructor(CancellationToken cancellationTok
         }
 
         private TLanguageService GetRequiredLanguageService<TLanguageService>(string language) where TLanguageService : ILanguageService
-        {
-#if CODE_STYLE
-            return _document.Project.Solution.Workspace.Services.GetExtendedLanguageServices(language).GetRequiredService<TLanguageService>();
-#else
-            return _document.Project.Solution.Services.GetRequiredLanguageService<TLanguageService>(language);
-#endif
-        }
+            => _document.Project.Solution.Workspace.Services.GetExtendedLanguageServices(language).GetRequiredService<TLanguageService>();
```

Comment:
the call to GetExtendedLanguageServices already defers to GetRequiredLanguageService if not in the code-style layer.  so we can simplify all of this.